From 9460f57d8e22d19ede8f778f10ab35dd6a4ee617 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Wed, 3 Apr 2013 13:25:38 -0500
Subject: [PATCH 11/13] ARM: OMAP: am335x: musb use 250 for power

Issue first noticed by: Randy Rodes <randyrodesnn@gmail.com>
https://groups.google.com/d/msg/beagleboard/qzlwO9ldULE/kXzFAY9rv8cJ

According to the docs, the power value is only 8 bits, to specify 500mA
this needs to be set to a value of 250: value * 2 = XmA

Documentation/devicetree/bindings/usb/omap-usb.txt

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 arch/arm/boot/dts/am335x-bone-common.dtsi | 2 +-
 arch/arm/boot/dts/am335x-tester.dts       | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/am335x-bone-common.dtsi b/arch/arm/boot/dts/am335x-bone-common.dtsi
index 90178e2..89ec6f2 100644
--- a/arch/arm/boot/dts/am335x-bone-common.dtsi
+++ b/arch/arm/boot/dts/am335x-bone-common.dtsi
@@ -478,6 +478,6 @@
 
 &usb_otg_hs {
 	interface_type = <1>;
-	power = <500>;
+	power = <250>;
 	status = "okay";
 };
diff --git a/arch/arm/boot/dts/am335x-tester.dts b/arch/arm/boot/dts/am335x-tester.dts
index 4a5ae29..c6ac8bf 100644
--- a/arch/arm/boot/dts/am335x-tester.dts
+++ b/arch/arm/boot/dts/am335x-tester.dts
@@ -478,6 +478,6 @@
 
 &usb_otg_hs {
 	interface_type = <1>;
-	power = <500>;
+	power = <250>;
 	status = "okay";
 };
-- 
1.8.1.4

